src/Orleans.Core.Abstractions/CodeGeneration/KnownAssemblyAttribute.cs

```diff
@@ -34,6 +34,7 @@ public KnownAssemblyAttribute(string assemblyName)
         /// serializable.
         /// </summary>
         /// <remarks>This is equivalent to specifying <see cref="KnownTypeAttribute"/> for all types.</remarks>
+        [Obsolete("This property is no longer supported. Serializers will be generated for all accessible types.")]
```

Comment:
What's the reason for this change?

Reply:
It's simpler and less error prone to just consider all types in the few assemblies we're processing

Reply:
But could be a lot more expensive now that runtime codegen is back, right? I'm not pushing back, just trying to understand

Reply:
It could be, since we need to perform accessibility checks for more types now. We cache those checks, though, so it shouldn't be an issue.